api/envoy/api/v2/cds.proto

```diff
@@ -376,6 +376,12 @@ message Cluster {
     // host selected but Y having 100, then a lot more load is being dumped on the single host in X
     // than originally anticipated in the load balancing assignment delivered via EDS.
     bool locality_weight_aware = 4;
+
+    // When used with locality_weight_aware, scales the weight of each locality by the ratio
+    // of hosts in the subset vs hosts in the original subset. This aims to even out the load
```

Comment:
Can you extend https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing.html with a worked example? That would make it clear what this does to the user.